docs/core/deploying/creating-nuget-packages.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Creating a NuGet Package with Cross Platform Tools | Microsoft Docs
-description: Creating a NuGet Package with Cross Platform Tools
+description: Shows how to create a NuGet package with the 'dotnet pack' command.
```

Comment:
I was reading a bit about best practices for descriptions. Two things I'd like to highlight from that:
- its main purpose is to get the visitor from Google to click your link
- It should be actionable, in an active voice.

So, in this example, I'd change from "Shows how" to "Learn how". Review the others to see if they're following that and use words like Learn, Understand, etc.